README.md

```diff
@@ -844,24 +844,17 @@ If you want to embed Regal in another project or product, please reach out!
 
 ## Roadmap
 
-The Regal project roadmap is subject to change, but these are some of the features we're planning to work on
-in the near future:
-
-### Linter
-
-- [x] Full support for both OPA 1.0 policies and older versions of Rego
-- [ ] Allow remediation of more `style` category rules using the `regal fix` command
-- [ ] Add [unused-rule](https://github.com/StyraInc/regal/issues/358) linter
-- [x] Add [unused-output-variable](https://github.com/StyraInc/regal/issues/60) linter
-
-### Language Server
-
-- [ ] Make "Check on save" unnecessary by allowing diagnostics to include
-      [compilation errors](https://github.com/StyraInc/regal/issues/745)
-- [x] Add Code Lens to "Evaluate" any rule or package (VS Code only, initially)
-- [ ] Implement [Signature Help](https://github.com/StyraInc/regal/issues/695) feature
-
-The roadmap is updated when all the current items have been completed.
+The current Roadmap items are all related to the preparation for
+[Regal 1.0](https://github.com/StyraInc/regal/issues/979):
+
+- [ ] [CLI: Remove the json output format in favor of sarif (#1553)](https://github.com/StyraInc/regal/issues/1553)
```

Comment:
Let’s drop this, and instead deprecate and hide

Reply:
Have updated now